persistence-model/src/main/scala/hmda/persistence/serialization/institutions/InstitutionProtobufSerializer.scala

```diff
@@ -0,0 +1,29 @@
+package hmda.persistence.serialization.institutions
+
+import akka.serialization.SerializerWithStringManifest
+import hmda.persistence.messages.events.institutions.InstitutionEvents.{ InstitutionCreated, InstitutionModified }
+import hmda.persistence.model.serialization.InstitutionEvents.{ InstitutionCreatedMessage, InstitutionModifiedMessage }
+import hmda.persistence.serialization.institutions.InstitutionProtobufConverter._
+
+class InstitutionProtobufSerializer extends SerializerWithStringManifest {
+  override def identifier: Int = 1000
+
+  override def manifest(o: AnyRef): String = o.getClass.getName
+
+  final val InstitutionCreatedManifest = classOf[InstitutionCreated].getName
+  final val InstitutionModifiedManifest = classOf[InstitutionModified].getName
+
+  override def toBinary(o: AnyRef): Array[Byte] = o match {
```

Comment:
These are used by `Akka Persistence` when storing / reading objects to / from the journal. They are not called directly from our code. It doesn't surprise me that `IntelliJ` gets confused.